operations/index.md

```diff
@@ -146,9 +148,11 @@ Procedures required for a full power off of an HPE Cray EX system.
 Procedures required for a full power on of an HPE Cray EX system.
 
   * [System Power On Procedures](power_management/System_Power_On_Procedures.md)
+
+Additional links to *power on sub procedures* provided for reference:
```

Comment:
Instead of adding this line, we can just indent the bullets under this line. This is how the other sections of the index have shown subtopics.

Reply:
Fixed and updated files.